tests/functional/spec/components/checkbox/checkbox_numeric_detail_answers.spec.js

```diff
@@ -0,0 +1,116 @@
+const helpers = require('../../../helpers');
+
+const CheckboxNumericDetailPage = require('../../../generated_pages/checkbox_numeric_detail_answers/checkbox-numeric-detail.page');
+const SummaryPage = require('../../../generated_pages/checkbox_numeric_detail_answers/summary.page');
+
+describe('Checkbox with a numeric "detail_answer" option', function() {
+
+  const checkbox_schema = 'test_checkbox_numeric_detail_answers.json';
+
+  it('Given a numeric detail answer options are available, When the user clicks an option, Then the detail answer input should be visible.', function() {
+    // Given
+    return helpers.openQuestionnaire(checkbox_schema).then(() => {
+      return browser
+      // When
+        .click(CheckboxNumericDetailPage.other())
```

Comment:
This is also executed for each test so can be moved into the beforeEach method available on describe